content/en/docs/concepts/secure-deployment.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Securely deploying Notation"
+description: "Best practices for securely deploying Notation"
+type: docs
+weight: 2
+---
+
+A deployment of Notation, specifically the `notation` CLI, is used for signing and verifying container images. The `notation` CLI is usually deployed on a development computer or using a CI/CD pipeline, such as GitHub Actions.
+
+A deployment has several sensitive components:
+
+- Security policies
+- Certificates
+- Trust stores
+- Plugins
+
+It is strongly recommended that you follow the best practices in this document to secure your deployment of Notation.
+
+## Secure filesystem access
```

Comment:
I think we should get more into details like:
- Only Notation user having access to those folders (or sudo)

Also, we need to explain what are the risks of somebody accessing any of those folders. Like replacing plugin with malicious one, changing the policy to allow images signed with malicious key, adding untrusted certs etc.

Reply:
fixed